### PR TITLE
fix: broken community links

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: docs/community/contributing_guide.mdx [hunk 1]
```diff
@@ -60,8 +60,8 @@ own ideas, and experiences.
 
 Other than pull requests and GitHub issues, the best way to contact us is to
 join us at the [CNCF Slack][cncf-slack] under the [**#oras** channel][slack].
-You can find the steps in the [community
-resources](/docs/community/community_resources#joining-the-slack-channel).
+You can find the steps in the
+[community resources](/community/community_resources.mdx#joining-the-slack-channel).
```
Comment (qweeah) [inline docs/community/contributing_guide.mdx:64]: Should it be
```suggestion
[community resources](/next/community/community_resources/#joining-the-slack-channel).
```

Comment (TerryHowe) [reply docs/community/contributing_guide.mdx:64]: This one works
```
* mdx [community resources](/community/community_resources.mdx#joining-the-slack-channel).
```

Comment (TerryHowe) [reply docs/community/contributing_guide.mdx:64]: Minimized the changes to just fixing the links

Comment (qweeah) [reply docs/community/contributing_guide.mdx:64]: WellI am not a frontend expert but `mdx` sounds like a thing that only exist before building out the web page, that's why I am commenting to get it removed from the link. 

Just FYI, I tried `/community/community_resources/#joining-the-slack-channel` and it also works. I am okay with both choices.

**Review by qweeah — APPROVED** [review]
LGTM but IANAM

**Review by asmitbm — APPROVED** [review]
lgtm

**Pull request merged**